### ISSUE TITLE
[Feature Request] TWL NAND support

### ISSUE THREAD

**Issue by Nolij** [Bas25/HorizonMod#1]
This is a long shot - pretty sure it should be an entirely different project and would be extremely difficult if not impossible, but on the off chance it's not, I'd like to request support for streaming TWL NAND (DS/i games).

**Comment by Meganium97** [comment]
No, since the 3ds enters a different state when loading DS(i) games, even through TWL nand. You're going to need a fairly expensive dedicated capture card to do that.

**Comment by Nolij** [comment]
Could an approach similar to TWPatch be used? I'm fairly certain some of the injection method could be re-used.

**Comment by Meganium97** [comment]
That's not my question to answer.
